shell/platform/windows/flutter_window_unittests.cc

```diff
@@ -377,5 +378,17 @@ TEST(FlutterWindowTest, OnWindowRepaint) {
   win32window.InjectWindowMessage(WM_PAINT, 0, 0);
 }
 
+TEST(FlutterWindowTest, OnThemeChange) {
+  MockFlutterWindow win32window;
+  MockWindowBindingHandlerDelegate delegate;
+  win32window.SetView(&delegate);
+
+  ON_CALL(win32window, GetHighContrastEnabled()).WillByDefault(Return(true));
+  EXPECT_CALL(win32window, GetHighContrastEnabled()).Times(1);
```

Comment:
I'd skip this line, since it feels a bit like we're testing that the implementation _is_ the current implementation. 

The primary thing we care about is that when we get a theme changed event we update the contrast enabled state if Windows tells us that high contrast mode is enabled. To enable that, we want to test that `UpdateHighContrastEnabled` is set true. Since that method happens to call GetHighContrastEnabled, we need to mock that out with the ON_CALL above, but conceptually, it's not fundamental to the test that that method gets called.